react-big-calendar/index.d.ts

```diff
@@ -26,10 +26,10 @@ declare module 'react-big-calendar' {
         step?: number;
         rtl?: boolean;
         eventPropGetter?: (event: Object, start: stringOrDate, end: stringOrDate, isSelected: boolean) => void;
-        titleAccessor?: string;
-        allDayAccessor?: boolean;
-        startAccessor?: stringOrDate;
-        endAccessor?: stringOrDate;
+        titleAccessor?: string | ((row: Object) => string);
+        allDayAccessor?: string | ((row: Object) => boolean);
```

Comment:
the reason for not using Object is because it will allow to use other incompatible types like for instance `boolean` or 'number` and allow for runtime errors.
From examples you provided I see this is used as a callback so maybe it's not that important, but anyway it's better to be as safe as possible and follow best practices